Issue #275: Add basic README.md (#277)
PolicyEngine/policyengine-app: README.md

# Welcome :wave: !
This is the official web app of the PolicyEngine project. <br/><br/>
![PolicyEngineWebApp](https://user-images.githubusercontent.com/117248915/221730075-b368a1df-d0d4-455c-8768-329e1eb9318e.png)

# Getting started

## Prerequisites
We recommend that you use the latest Node version 19. To easily manage your node versions, get the Node Version Manager [nvm](https://github.com/nvm-sh/nvm) and then do

```
nvm install 19 && nvm use 19
```

## Setup
1. Clone the repo
```
https://github.com/PolicyEngine/policyengine-app.git
```
2. Install dependencies

```
npm install
```
3. Start a server on localhost to see your changes
```
npm start
```
Now you're ready to start developing!

To test your changes, do
```
npm test
```

# Contributing
If you're new to the project, go to Issues and search for good first issues `label: "good first issue"`.

> Reach out to @nikhilwoodruff and @MaxGhenis to be added as a collaborator so that you can make new PRs on the repo.

Reviewer: Anyone can make a PR.
```suggestion
Reach out to @nikhilwoodruff and @MaxGhenis to be added as a collaborator so that you can assign yourself to issues.
```

Please try to be detailed in your PRs about the changes you made and why you made them. You may find yourself looking back at them for reference in the future, or needing insight about someone else's changes. Save yourself a conversation and write it all in the PR!

Here are some [best practices](https://deepsource.io/blog/git-best-practices/) for using Git.

> It is recommended that you try to add tests for any new features or bug-fixes you add, so we can gradually build up the code coverage. We use [Jest](https://jestjs.io/docs/tutorial-react) and the [React Testing Library](https://github.com/testing-library/react-testing-library). You can run your tests locally with `npm test` and they will be run again in the PR pre-flight.

Reviewer: ```suggestion
We recommend, but do not yet require, that you add tests for any new features or bug-fixes you add, so we can gradually build up the code coverage. We use [Jest](https://jestjs.io/docs/tutorial-react) and the [React Testing Library](https://github.com/testing-library/react-testing-library). You can run your tests locally with `npm test` and they will be run again in the PR pre-flight.
```

> When you're ready for review, add a contributor as a reviewer.

Reviewer: ```suggestion
When you're ready for review, switch the PR from `Draft` to `Ready for review` and add a contributor as a reviewer.
```
